edb/testbase/server.py

```diff
@@ -444,7 +444,8 @@ async def init_cluster(
         )
         destroy = False
 
-    if await cluster.get_status() == 'not-initialized':
+    pg_cluster = await cluster._get_pg_cluster()
+    if await pg_cluster.get_status() == 'not-initialized':
```

Comment:
This code is a bit weird -- why didn't `cluster.get_status() == 'not-initialized'` work as expected?

Reply:
Because that `get_status` will try to do a bunch of other things that break in the case where we are trying to reuse an already created database

Reply:
Hm, maybe we should refactor that code. Fine for now, but this is brittle.